interface/super/edit_globals.php

```diff
@@ -50,7 +54,7 @@ function checkCreateCDB(){
       $GLOBALS[$globalsrow['gl_name']] = $globalsrow['gl_value'];
     }
     $directory_created = false;
-  if( !empty($GLOBALS['document_storage_method']) ) {
+  if($GLOBALS['document_storage_method'] != 0){
```

Comment:
Well, it's all good by me.  I wuz just axin'.  This is ready to go as far as I can see. @tmccormi